.eslintrc.json

```diff
@@ -9,6 +9,7 @@
                 "@nrwl/nx/enforce-module-boundaries": [
                     "error",
                     {
+                        "allowCircularSelfDependency": true,
```

Comment:
Nx recommend not to use mapped paths in the apps and only use relative paths but our `dotcms-ui` app is so big that we can use `../../../../../../`